src/main/java/ch/admin/bag/covidcode/authcodegeneration/api/AuthorizationCodeVerifyResponseDtoWrapper.java

```diff
@@ -0,0 +1,34 @@
+package ch.admin.bag.covidcode.authcodegeneration.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AuthorizationCodeVerifyResponseDtoWrapper {
+
+  private AuthorizationCodeVerifyResponseDto swissCovidAccessToken;
+  private AuthorizationCodeVerifyResponseDto notifyMeAccessToken;
+
+  public AuthorizationCodeVerifyResponseDtoWrapper(AuthorizationCodeVerifyResponseDto swissCovidAccessToken, AuthorizationCodeVerifyResponseDto notifyMeAccessToken) {
+    this.swissCovidAccessToken = swissCovidAccessToken;
+    this.notifyMeAccessToken = notifyMeAccessToken;
+  }
+
+  public AuthorizationCodeVerifyResponseDtoWrapper() {}
+
+
+  public AuthorizationCodeVerifyResponseDto getSwissCovidAccessToken() {
+    return swissCovidAccessToken;
+  }
+
+  public void setSwissCovidAccessToken(AuthorizationCodeVerifyResponseDto swissCovidAccessToken) {
+    this.swissCovidAccessToken = swissCovidAccessToken;
+  }
+
+  public AuthorizationCodeVerifyResponseDto getNotifyMeAccessToken() {
```

Comment:
I'd suggest to rename this to  `getCheckInUserUploadAccessToken`.

Reply:
done